static/app/components/charts/scatterChart.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react';
+
+import {Series} from 'sentry/types/echarts';
+
+import ScatterSeries from './series/scatterSeries';
+import BaseChart from './baseChart';
+
+type ChartProps = Omit<React.ComponentProps<typeof BaseChart>, 'css'>;
+
+export type ScatterChartSeries = Series;
+
+type Props = Omit<ChartProps, 'series'> & {
+  series: ScatterChartSeries[];
+};
+
+function ScatterChart({series, ...props}: Props) {
+  return (
+    <BaseChart
+      {...props}
+      xAxis={{
+        // do not want the axis pointer when working with scatter charts
+        axisPointer: {
+          show: false,
+          ...props.xAxis?.axisPointer,
+        },
+        ...props.xAxis,
+      }}
+      series={series.map(({seriesName, data, ...options}) =>
+        ScatterSeries({
+          name: seriesName,
+          data: data.map(({name, value}) => [name, value, 'stuff']),
+          ...options,
+          animation: false,
+          animationThreshold: 1,
```

Comment:
Either of these options required with `animation: false`?  Also since they are after ...options you are forcing animation off no matter the options setting.. intended?

Reply:
Hm... that's a good question, I took these options from the other charts so I didn't look into them much but it looks like `animation: false` disables all animations so the following options don't actually do anything at all. I'll remove them.